### PR TITLE
 (SIMP-1354) Add a useful README.md

### PR DESCRIPTION
**Pull request by silug**
This adds a README.md based on Puppet strings along with additional small documentation fixes:

* Fix documentation for logconfig parameter.
* Spelling fix.
* Document firewall parameter.

SIMP-1354 #close

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,137 @@
-## Work in Progress
+[![License](http://img.shields.io/:license-apache-blue.svg)](http://www.apache.org/licenses/LICENSE-2.0.html) [![Build Status](https://travis-ci.org/simp/pupmod-simp-ntpd.svg)](https://travis-ci.org/simp/pupmod-simp-ntpd) [![SIMP compatibility](https://img.shields.io/badge/SIMP%20compatibility-6.*-orange.svg)](https://img.shields.io/badge/SIMP%20compatibility-6.*-orange.svg)
 
-Please excuse us as we transition this code into the public domain.
+# ntpd
```
Comment (nick-markowski) [inline README.md]: pupmod-simp-ntpd

File: README.md [hunk 2]
```diff
@@ -1,5 +1,137 @@
...
+
+A standard `ntpd.conf` restrict append rule (`notrust`, etc...)
+
+* firewall (`Boolean`) *(defaults to: `simplib::lookup('simp_options::firewall', { 'default_value' => false})`)*
+
+If enabled, allow connections from `trusted_nets`
+
+## Limitations
+
+SIMP Puppet modules are generally intended to be used on a Red Hat Enterprise
```
Comment (nick-markowski) [inline README.md]: You should pull this from metadata.json

File: README.md [hunk 3]
```diff
@@ -1,5 +1,137 @@
...
+Linux-compatible distribution.
+
+## Development
+
+Please read our [Contribution Guide](https://simp-project.atlassian.net/wiki/display/SD/Contributing+to+SIMP)
```
Comment (nick-markowski) [inline README.md]: This and the developer guide should point to the readthedocs pages

File: README.md (hunk 2 above)
Comment (silug) [reply README.md]: That was directly pulled from another module ([pupmod-simp-named](https://github.com/simp/pupmod-simp-named#limitations)).  Can you point me to an example that does this right?

File: README.md (hunk 1 above)
Comment (silug) [reply README.md]: Fixed.

File: README.md (hunk 2 above)
Comment (silug) [reply README.md]: I pulled the language from [pupmod-simp-at](https://github.com/simp/pupmod-simp-at/#limitations).

File: README.md (hunk 3 above)
Comment (silug) [reply README.md]: Fixed.

File: README.md [hunk 4]
```diff
@@ -1,5 +1,138 @@
...
+
+```
+servers => {
+  'time.local.net' => ['iburst','minpoll 4', 'prefer'],
+  # This one will just use $default_options
+  'time.other.net' => []
+}
+```
+
+* stratum (`Integer[0]`) *(defaults to: `2`)*
```
Comment (nick-markowski) [inline README.md]: The formatting here makes it hard to find the list of parameters.  You might consider bolding them.

Comment (judyj) [reply README.md]: typo-ed monitoring in line 104

Comment (silug) [reply README.md]: @judyj Fixed.

Comment (silug) [reply README.md]: @nick-markowski The parameters should have been `code`, but only the first one was.  They're now **`bold code`**.

**Pull request merged**